### PR TITLE
Adds Grupo Amapola Uniforms

### PR DESCRIPTION
**Pull request by lavillastrangiato**
* Adds uniforms and hats for the Grupo Amapola.
* Sprites made by me (Lavillastrangiato), recoloured/modified from the Sol army uniforms.

To-do:
- [x] Make the rolled sleeves and the rolled-down suit sprites show up
- [x] Figure out why the hats aren't showing in the loadout

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!wip

File: aurorastation.dme [hunk 1]
```diff
@@ -3725,12 +3725,12 @@
 #include "code\ZAS\Zone.dm"
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
+#include "interface\skin.dmf"
 #include "interface\fonts\fonts_datum.dm"
 #include "interface\fonts\grand_9k.dm"
 #include "interface\fonts\pixellari.dm"
 #include "interface\fonts\spess_font.dm"
 #include "interface\fonts\tiny_unicode.dm"
-#include "interface\skin.dmf"
```
Comment (Generalcamo) [inline aurorastation.dme:3733]: This shouldn't be included

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: code/modules/client/preference_setup/loadout/items/factions.dm [hunk 2]
```diff
@@ -474,6 +476,14 @@
 	faction = "Private Military Contracting Group"
 	slot = slot_w_uniform
 
+/datum/gear/faction/grupo_amapola_officer
+ 	display_name = "grupo amapola officer cap"
+	path = /obj/item/clothing/head/misc/grupo_amapola
+	flags = GEAR_HAS_DESC_SELECTION
+	allowed_roles = list("Head of Security")
+	faction = "Private Military Contracting Group"
```
Comment (Generalcamo) [inline code/modules/client/preference_setup/loadout/items/factions.dm:484]: The head of security cannot be part of the PMCG.

Comment (lavillastrangiato) [reply code/modules/client/preference_setup/loadout/items/factions.dm:484]: stellar corporate conglomerate can take any faction gear

File: aurorastation.dme (hunk 1 above)
Comment (lavillastrangiato) [reply aurorastation.dme:3733]: think i got it

Comment (listerla) [inline aurorastation.dme:3733]: To specify, the mistake here is moving the line 3733 to line 3728; that, specifically, should be reverted.

Comment (lavillastrangiato) [reply aurorastation.dme]: thank you! lly fixed it, happily

**Comment by lavillastrangiato** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**